### PR TITLE
Icon update

### PR DESCRIPTION
**Pull request by akshaisarma**
Addresses #51, #52, #59

### CODE REVIEW

File: app/templates/components/cells/schema-name-entry.hbs [hunk 1]
```diff
@@ -9,9 +9,9 @@
 {{#if row.hasEnumerations}}
   <span class="schema-enumeration-caret">
     {{#if row.expanded}}
-      <i class="expanded-caret glyphicon glyphicon-menu-down"></i>
+      <i class="expanded-caret fa fa-chevron-down"></i>
```
Comment (hbxie) [inline app/templates/components/cells/schema-name-entry.hbs]: change to {{ fa-icon down class="expanded-caret"}}?

File: app/templates/components/cells/schema-name-entry.hbs [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
     {{else}}
-      <i class="expand-caret glyphicon glyphicon-menu-right"></i>
+      <i class="expand-caret fa fa-chevron-right"></i>
```
Comment (hbxie) [inline app/templates/components/cells/schema-name-entry.hbs]: ditto

File: app/templates/components/cells/schema-name-entry.hbs (hunk 1 above)
Comment (akshaisarma) [reply app/templates/components/cells/schema-name-entry.hbs]: Done

File: app/templates/components/cells/schema-name-entry.hbs (hunk 2 above)
Comment (akshaisarma) [reply app/templates/components/cells/schema-name-entry.hbs]: Done

**Pull request merged**